### DISCUSSION TITLE
Notifications?

### DISCUSSION THREAD

**Discussion by realbaylen** [MusicPlayerDaemon/MPD#2024] [Q&A] [answered]
https://wiki.archlinux.org/title/ncmpcpp#Notification_on_song_change

Is there a way this can be accomplished on ncmpc?

**Comment by MaxKellermann** [comment, marked as answer]
As the maintainer of ncmpc, I consider this a misfeature that does not belong in a general-purpose client. Imagine you happen to have two instances of ncmpc running - you'll get each notification twice; and when you have closed all ncmpc instances, you'll cease to get any notifications. With the MPD architecture, you better use a specialized non-interactive client that launches as part of your login session; for example there's "mpdcron".

**Reply by kingosticks** [reply]
Examples at https://wiki.archlinux.org/title/Music_Player_Daemon/Tips_and_tricks#Notifications

**Reply by realbaylen** [reply]
That makes a lot of sense. Thanks!
